### DISCUSSION TITLE
Typo on screen - No Automations vs. Not Automations

### DISCUSSION THREAD

**Discussion by bduplanty** [stacks-task-manager/stacks2#46] [Bugs]
Language issue?  How would you like us to report this?  As a bug or in discussion?
![Screenshot 2023-06-18 at 11 58 14 AM](https://github.com/stacks-task-manager/stacks2/assets/33303709/867a8f5e-7744-4f8e-8c8e-76d778147aa5)

**Comment by synapse** [comment]
Thanks @bduplanty, this should be fixed in the next release
